trace/collector.go

```diff
@@ -121,6 +122,11 @@ func (pc *PacketCountCollector) Process(t akinet.ParsedNetworkTraffic) error {
 			DstPort:      t.DstPort,
 			HTTPRequests: 1,
 		})
+		// only count the capture as success if we see total.Requests > 0 && total.HTTPResponses > 0
```

Comment:
I have added this check since the backend has the following condition while inferring an error from the capture stats
```
// Modify the response with an inferred capture error.
func inferCaptureError(resp *rest.ClientPacketCaptureStats) *rest.ClientPacketCaptureStats {
	// If there's already a client-reported error, use that.
	If resp.Error != nil {
		return resp
	}

	// ... Other checks
	
	// Did we get complete HTTP traffic (in both directions)?
	If summary.Total.HTTPRequests > 0 && summary.Total.HTTPResponses > 0 {
		// no error
		return resp
	}

	// ... Other checks

	// Unknown reason we can't parse it.
	return setError(rest.InferredError_Unparsable)
}
```